wiki-jpa-dao/src/main/resources/db/changelog/wiki.db.changelog-1.0.0.xml

```diff
@@ -51,11 +52,11 @@
         <constraints nullable="false"/>
       </column>
       <column name="OWNER" type="NVARCHAR(200)"/>
-      <column name="CREATED_DATE" type="TIMESTAMP" defaultValueDate="${now}">
-        <constraints nullable="false"/>
+      <column name="CREATED_DATE" type="TIMESTAMP">
+        <constraints nullable="true"/>
       </column>
-      <column name="UPDATED_DATE" type="TIMESTAMP" defaultValueDate="${now}">
-        <constraints nullable="false"/>
+      <column name="UPDATED_DATE" type="TIMESTAMP">
+        <constraints nullable="true"/>
```

Comment:
Isn't there any impact to set it as nullable ? If the value is null, do we use the created date ?